openrouteservice/src/main/java/org/heigit/ors/routing/RoutingProfileManager.java

```diff
@@ -325,7 +325,9 @@ public RouteResult matchTrack(MapMatchingRequest req) throws Exception {
             throw new InternalServerException(RoutingErrorCodes.INVALID_PARAMETER_VALUE, "Alternative routes algorithm does not support more than two way points.");
         }
 
-        ExtraInfoProcessor extraInfoProcessor = null;
+        int numberOfExpectedExtraInfoProcessors = req.getSearchParameters().getAlternativeRoutesCount();
```

Comment:
Could these two lines be combined/made a bit easier to read?

Reply:
i don't really know if it made it easier to read...